### PR TITLE
update verbosity, add more log levels

### PR DESCRIPTION
**Pull request by ngtuna**
Turns value of `--verbose` to string and add more choices: debug, info, warning, error, fatal, panic

### CODE REVIEW

**Comment by andresmgot** [comment]
do you have messages for all those levels?

**Comment by ngtuna** [comment]
@andresmgot I added a commit for describing log levels and also remove panic, fatal levels which are not really helpful and less used. PTAL

File: cmd/root.go [hunk 1]
```diff
@@ -71,16 +71,24 @@ Find more information at https://github.com/kubeapps/kubeapps.`,
 }
 
 func init() {
-	RootCmd.PersistentFlags().CountP("verbose", "v", "Increase verbosity.")
+	RootCmd.PersistentFlags().StringP("verbose", "v", "warning", fmt.Sprint("Set log level: debug, info, warning, error.\n"+
```
Comment (andresmgot) [inline cmd/root.go]: the usual value for the default is `info`, isn't it?

Comment (ngtuna) [reply cmd/root.go]: yes default to `info` makes sense. The `warning` was there for a while

**Pull request merged**